tbb/src/DataFlowGraph.cxx

```diff
@@ -1,10 +1,10 @@
-
-
 #include "WireCellTbb/DataFlowGraph.h"
 
 #include "WireCellUtil/Type.h"
 #include "WireCellUtil/NamedFactory.h"
 
+#include <tbb/task_scheduler_init.h>
```

Comment:
This header seems to be gone from TBB 2021.1.1.  @knoepfel I wonder how you were able to build.  Maybe a second TBB include was in the build environment?

The "tbbrevamp" doc says `tbb::task_arena` or `tbb::global_control` are the replacement.  You have added the latter so we are in the right direction.  Is it clear how to make use of this replacement?

Reply:
Looking through TBB includes and source, I'm not finding something equivalent to `automatic`.  I'm not even sure what that translated to.  I probably copied from some random example.

I'm tempted to make the default be simply `1`.  Any worries?